frontend/src/renderer/components/Process/Sorter.svelte

```diff
@@ -4,6 +4,7 @@
   import { dndzone } from "svelte-dnd-action";
   import { flip } from "svelte/animate";
   import ImageBubble from "./ImageBubble.svelte";
+  import { isEmpty, countBy } from 'lodash';
```

Comment:
do we fr need lodash for an is empty check ?

Reply:
Yessir, lodash is amazing and has the most useful set of functions you could ever want. Guarantee I'm gonna end up using it in all of my changes

Reply:
In all seriousness, it's the most useful JS library in existence